pkg/cloud/provider.go

```diff
@@ -35,9 +35,11 @@ const (
 
 	WorkerTimeout = time.Duration(2 * time.Hour)
 
-	defaultCloudName = "inCluster"
+	DefaultCloudName = "_inCluster"
 )
 
+var DefaultNamespace = "default"
```

Comment:
Define `DefaultNamespace` as const.

Reply:
no , DefaultNamespace will be assigned a new value while [init cloud](https://sourcegraph.com/github.com/zhujian7/cyclone@20f063851c30306af3b476c20ae8cbd8c343233f/-/blob/pkg/cloud/provider.go#L77).